packages/wonder-blocks-layout/util/types.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+export type MediaSize = "small" | "medium" | "large";
+
+export type MediaQuery = "all" | "mdOrSmaller" | "mdOrLarger" | MediaSize;
+
+// eslint-disable-next-line flowtype/require-exact-type
```

Comment:
Can you add a comment on why we're suppressing this? I think it will help others when they come to this code.

Reply:
Since this is now exact, isn't this lint suppression redundant?

Reply:
Thanks for catching that.